articles/azure-fluid-relay/how-tos/validate-document-creator.md

```diff
@@ -1,26 +1,26 @@
+When you create a container in Azure Fluid Relay, the JWT provided by the [ITokenProvider](https://fluidframework.com/docs/apis/azure-client/itokenprovider-interface) for the creation request can only be used once. After creating a container, the client must generate a new JWT that contains the document ID (which is really the container ID) provided by the service at creation time. If an application has an authorization service that manages container access control, it will need to know who created a container with a given ID in order to authorize the generation of a new JWT for access to that container.
 
-## Inform an Authorization Service when a document is Created
+## Inform an Authorization Service when a Container is Created
 
-An application can tie into the document creation lifecycle by implementing a public [documentPostCreateCallback()](https://fluidframework.com/docs/apis/azure-client/itokenprovider-interface#documentpostcreatecallback-methodsignature) method in its `TokenProvider`. This callback will be triggered directly after creating the document, before a client requests the new JWT it needs to gain read/write permissions to the document that was created.
+An application can tie into the container creation lifecycle by implementing a public [documentPostCreateCallback()](https://fluidframework.com/docs/apis/azure-client/itokenprovider-interface#documentpostcreatecallback-methodsignature) method in its `TokenProvider`. (The name of this function can be confusing. It is really a callback for post *container* creation.) This callback will be triggered directly after creating the container, before a client requests the new JWT it needs to gain read/write permissions to the container that was created.
 
-The `documentPostCreateCallback()` receives two parameters: 1) the ID of the document that was created and 2) a JWT signed by the service with no permission scopes. The authorization service can verify the given JWT and use the information in the JWT to grant the correct user permissions for the newly created document.
+The `documentPostCreateCallback()` receives two parameters: 1) the ID of the container that was created (also called the "document ID") and 2) a JWT signed by the service with no permission scopes. The authorization service can verify the given JWT and use the information in the JWT to grant the correct user permissions for the newly created container.
 
-### Create an endpoint for your document creation callback
+### Create an endpoint for your container creation callback
```

Comment:
The other headings are mostly capitalized. Should this one do the same? I.e.
```suggestion
### Create an Endpoint for Your Container Creation Callback
```

Reply:
Current Microsoft Style Guide says to use sentence-casing for section titles, not title-casing. Also, we use sentence-casing in other articles in the Azure docs.

Reply:
Ah, okay. Should we update the other headings to use sentence case then? Not really in the scope of this PR, so feel free to close with no action :)